### PR TITLE
More gender neutral translation

### PR DESCRIPTION
**Pull request by sha-265**
More gender neutral Hebrew translation

### CODE REVIEW

**Review by tulir — COMMENTED** [review]
Looks like this got a merge conflict at some point

**Comment by mtatko** [comment]
@jonatan1609, would you like to review this translation as you wrote the original? Thanks in advance!

**Comment by sha-265** [comment]
@mtatko, FWIW @jonatan1609 liked this pull request https://github.com/maunium/dontasktoask.com/pull/68#issue-1038829360

**Review by mtatko — APPROVED** [review]

**Comment by mtatko** [comment]
Waiting for merge on this one then

**Review by tulir — APPROVED** [review]

File: he/index.html [hunk 1]
```diff
@@ -58,27 +58,27 @@ <h1>אל תשאל אם אפשר לשאול. פשוט תשאל!</h1>
...
     </p>
     <p>
-      לסיכום, אל תשאל
+      לסיכום, אל תשאלו
       <em>"יש פה מומחי ג’אווה?"</em>,
-      שאל
+      שאלו
       <em>
       "יש לי בעיה X כשאני מנסה לעשות Y בשימוש בכלי Z. ניסיתי לפתור זאת באמצעות A, B, C אך ללא הצלחה.
       לא מצאתי תשובה רלוונטית בגוגל. יש למישהו רעיון איך לפתור את הבעיה?".
```
Comment (jonatan1609) [inline he/index.html:84]: I'd fix מישהו to מישהו/י as well at this line

File: he/index.html [hunk 2]
```diff
@@ -58,27 +58,27 @@ <h1>אל תשאל אם אפשר לשאול. פשוט תשאל!</h1>
 		..שזה פשוט עצלני. אם אתה לא מוכן לתת את ההשקעה הנדרשת כדי לפתור את הבעיה שלך, למה שאנחנו כן?
```
Comment (jonatan1609) [inline he/index.html:58]: אתה לא מוכן
can be changed to
אתם לא מוכנים

File: he/index.html [hunk 3]
```diff
@@ -15,7 +15,7 @@
 </head>
 <body dir="rtl">
   <main>
-    <h1>אל תשאל אם אפשר לשאול. פשוט תשאל!</h1>
+    <h1>אל תשאלו בשביל לשאול. פשוט תשאלו!</h1>
     <p>
       מידי פעם בקבוצות וצ’אטים באינטרנט, קופץ מישהו ושואל משהו בסגנון:
```
Comment (jonatan1609) [inline he/index.html:20]: קופץ/ת מישהו/י

**Review by jonatan1609 — COMMENTED** [review]
Some more changes.

**Pull request merged**